### PR TITLE
Add ksqlDB Grafana Dashboard, + ksqldb jmxexporter change to include key metrics

### PR DESCRIPTION
**Pull request by OoRobFuoO**
https://docs.ksqldb.io/en/latest/reference/metrics/#persistent-query-status

### CODE REVIEW

File: jmxexporter-prometheus-grafana/assets/prometheus/jmx-exporter/confluent_ksql.yml [hunk 1]
```diff
@@ -25,10 +26,19 @@ rules:
       id: "$3"
   # "io.confluent.ksql.metrics:type=_confluent-ksql-<cluster-id>ksql-engine-query-stats"
   # The below statement parses KSQL Cluster Name and adds a new label so that per cluster data is searchable.
-  - pattern: io.confluent.ksql.metrics<type=_confluent-ksql-(.+)ksql-engine-query-stats><>([^:]+)
+  - pattern: io.confluent.ksql.metrics<type=_confluent-ksql-(.+)_ksql-engine-query-stats><>([^:]+)
```
Comment (waliaabhishek) [inline jmxexporter-prometheus-grafana/assets/prometheus/jmx-exporter/confluent_ksql.yml:29]: I just ran the KSQLDB Server and the _ does not exist in the JMX data. Can you help revert it back ?
Eg:
io.confluent.ksql.metrics< type=_confluent-ksql-ksql-clusterksql-engine-query-stats><>ERROR-queries

File: jmxexporter-prometheus-grafana/assets/prometheus/jmx-exporter/confluent_ksql.yml [hunk 2]
```diff
@@ -25,10 +26,19 @@ rules:
...
     name: "ksql_ksql_engine_query_stats_$2"
     labels:
       ksql_cluster: $1
+  # "io.confluent.ksql.metrics:type=ksql-queries,status=_confluent-ksql-<cluser-id>_query_<query>
+  # The below statement parses KSQL query specific status
+  - pattern: "io.confluent.ksql.metrics<type=(.+), status=_confluent-ksql-(.+)_query_(.+)><>(.+): (.+)"
```
Comment (waliaabhishek) [inline jmxexporter-prometheus-grafana/assets/prometheus/jmx-exporter/confluent_ksql.yml:35]: Same for this JMX bean. _ does not exist in the JMX data. Can you help revert it back ? 
Eg: 
io.confluent.ksql.metrics:status=_confluent-ksql-ksql-clusterquery_CSAS_EN_WIKIPEDIA_GT_1_COUNTS_9,type=ksql-queries

File: jmxexporter-prometheus-grafana/assets/prometheus/jmx-exporter/confluent_ksql.yml [hunk 3]
```diff
@@ -25,10 +26,19 @@ rules:
...
+    value: 1
+    name: $1_status
```
Comment (waliaabhishek) [inline jmxexporter-prometheus-grafana/assets/prometheus/jmx-exporter/confluent_ksql.yml:37]: Could you update it to follow the naming convention -- componentName_beanName_typeName_attributeName ?
Eg:
io.confluent.ksql.metrics:status=_confluent-ksql-ksql-clusterquery_CSAS_EN_WIKIPEDIA_GT_1_COUNTS_9,type=ksql-queries": {
      "query-status": "RUNNING",
      "error-status": "NO_ERROR"
    }
would become :
ksql_ksql_metrics_ksql_queries_query_status
ksql_ksql_metrics_ksql_queries_error_status

File: jmxexporter-prometheus-grafana/assets/prometheus/jmx-exporter/confluent_ksql.yml (hunk 1 above)
Comment (OoRobFuoO) [reply jmxexporter-prometheus-grafana/assets/prometheus/jmx-exporter/confluent_ksql.yml:29]: fixed, the _ got removed by accident

File: jmxexporter-prometheus-grafana/assets/prometheus/jmx-exporter/confluent_ksql.yml (hunk 2 above)
Comment (OoRobFuoO) [reply jmxexporter-prometheus-grafana/assets/prometheus/jmx-exporter/confluent_ksql.yml:35]: fixed

File: jmxexporter-prometheus-grafana/assets/prometheus/jmx-exporter/confluent_ksql.yml (hunk 3 above)
Comment (OoRobFuoO) [reply jmxexporter-prometheus-grafana/assets/prometheus/jmx-exporter/confluent_ksql.yml]: updated to match the existing naming convention

**Comment by waliaabhishek** [comment]
@OoRobFuoO : I have updated the KSQLDB dashboard a bit and updated the KSQL YAML files to add KStream parsing rules. Please try out the PR and let me know if the updates work for you. Will merge the PR as soon as you confirm.

**Comment by OoRobFuoO** [comment]
Except the dashboard layout which I think is a subjective thing, the rest looks to good me.

The dashboard layout can be adjusted according to personal preference, so I'm okay with the rest of the change

**Review by waliaabhishek — APPROVED** [review]
LGTM !

**Pull request merged**